templates/base.html

```diff
@@ -76,131 +76,139 @@
-                            <li role="presentation">
-                                <a href="{% url 'home' %}">
-                                    <span class="glyphicon glyphicon-home" aria-hidden="true"></span>
-                                    {% trans "Home" %}
-                                </a>
-                            </li>
-                            {% course_menu %}
-                            {% if user.is_authenticated %}
-                            <li role="presentation">
-                                <a href="{% url 'profile' %}">
-                                    <span class="glyphicon glyphicon-user" aria-hidden="true"></span>
-                            	    {{ user.first_name }} {{ user.last_name }}
-                                </a>
-                            </li>
-                            <li role="presentation">
-                                <a href="{% url 'logout' %}">
-                                    <span class="glyphicon glyphicon-log-out" aria-hidden="true"></span>
-                            	    {% trans "Log out"%}
-                                </a>
-                            </li>
-                            {% else %}
-                            <li role="presentation">
-                                <a href="{% url 'login' %}">
-                                    <span class="glyphicon glyphicon-log-in" aria-hidden="true"></span>
-                                    {% trans "Log in" %}
-                                </a>
-                            </li>
-                            {% endif %}
-                        </ul>
+                                {% else %}
+                                <li role="presentation">
+                                    <a href="{% url 'login' %}">
+                                        <span class="glyphicon glyphicon-log-in" aria-hidden="true"></span>
+                                        {% trans "Log in" %}
+                                    </a>
+                                </li>
+                                {% endif %}
+                            </ul>
+                        </div>
                     </div>
-                </div>
-            </nav>
+                </nav>
+            </header>
 
-            <div class="site-content container-fluid" id="content">
-                {% site_alert %}
+            {% block container_opening_tag %}
+                <main class="site-content container-fluid" id="content">                
+            {% endblock %}
+            {% site_alert %}
                 {% include "_messages.html" %}
                 {% block content %}
                     <div class="error">
                         No content
                     </div>
                 {% endblock %}
-            </div>
+            {% block container_closing_tag %}
+                </main>                
+            {% endblock %}
```

Comment:
There seems to be a variety of indentation styles in this file alone (not indenting the content of template blocks vs. indenting). Since most of the changes in this file are indenting the the things put in new tags, you could fix indentation as well if you wish. The guidelines that are hoped for are.
- indent the content of template blocks
- use tabs to indent html-files and django templates